README.md

```diff
@@ -121,70 +100,164 @@
         <img alt="Prestashop" width="75" src="https://user-images.githubusercontent.com/1626923/156659137-d857ad24-f23f-4c80-ac56-4ef6c0721274.png" />
     </a>
     &nbsp;&nbsp;
-    <a title="nopCommerce" target="_blank" href="https://github.com/vuestorefront-community/nopcommerce">
-        <img alt="nopCommerce" width="75" src="https://user-images.githubusercontent.com/1626923/156659133-d047ea68-ec32-4ef1-aa7d-dc5ab88f5230.png" />
+    <a title="Shopware" target="_blank" href="https://github.com/vuestorefront/shopware-pwa">
+        <img alt="Shopware" width="75" src="https://user-images.githubusercontent.com/1626923/156659145-d98ea807-872c-42a9-8fdb-4c4f30a68895.png" />
     </a>
-<br />
     &nbsp;&nbsp;
-    <a title="kiboCommerce" target="_blank" href="https://github.com/vuestorefront-community/kibocommerce">
-        <img alt="kiboCommerce" width="75" src="https://user-images.githubusercontent.com/1626923/156659127-68c1442a-de03-46ab-863b-8dce2b5857d5.png" />
+<br />
+    <a title="Spree" target="_blank" href="https://github.com/vuestorefront/spree">
+        <img alt="Spree" width="75" src="https://user-images.githubusercontent.com/1626923/156659147-5576eb2f-4cf8-4610-8631-18d03e3f52de.png" />
     </a>
-  &nbsp;&nbsp;
+    &nbsp;&nbsp;
     <a title="Sylius" target="_blank" href="https://github.com/vuestorefront-community/sylius">
         <img alt="Sylius" width="75" src="https://user-images.githubusercontent.com/1626923/161141058-8f6a4a8e-2c6f-402e-b19b-892c7acec77f.png" />
-  </a>
+    </a>
+    &nbsp;&nbsp;
+    <a title="Vendure" target="_blank" href="https://github.com/vuestorefront/vendure">
+        <img alt="Vendure" width="75" src="https://user-images.githubusercontent.com/1626923/156659153-fafdeb94-a34e-4e9e-8e8a-0f03afde9741.png" />
+    </a>
     &nbsp;&nbsp;
         <img alt="Swell" width="75" src="https://user-images.githubusercontent.com/1626923/160184104-f4820722-3eff-4c8e-b33b-8134bc1dd78a.png" />
     &nbsp;&nbsp;
     <a title="WooCommerce" target="_blank" href="https://github.com/vuestorefront-community/woocommerce">
         <img alt="WooCommerce" width="75" src="https://user-images.githubusercontent.com/1626923/160184108-b54e8785-d44c-4023-957f-2fd40c286d88.png" />
   </a>
 <br /><br />
-  <a target="_blank" href="https://docs.vuestorefront.io/v2/integrations/">Learn more about available integrations</a>
+  <a target="_blank" href="https://docs.vuestorefront.io/v2/integrations/#ecommerce-platforms">Learn more about available eCommerce platforms</a>
+</p>
+
+## **Getting started** 🚀
+
+### **Step 1: Generate a new project** 🧰
+
```

Comment:
Might be good to call out that this is for the OS integrations, for the enterprise - they have to go through sales